tests/tools/onert_train/src/args.cc

```diff
@@ -199,6 +200,17 @@ void Args::Initialize(void)
     }
   };
 
+  auto loss_help_msg = []() -> std::string {
+    std::string res = "Loss type\n";
+
+    constexpr int printed_list[2] = {1, 2};
+    for (auto &i : printed_list)
+      res += std::to_string(i) + ": " + toString(static_cast<NNFW_TRAIN_LOSS>(i)) + "\n";
```

Comment:
### Motivation

For the help message, Let's reuse the related string defined in nnfw_utils.h
https://github.com/Samsung/ONE/pull/12445/files#r1448423248